lib/components/app-switcher/AppSwitcher.vue

```diff
@@ -13,8 +13,8 @@
           :style="{ 'background-color': hover || menuOpen ? '' : currentAppColor }"
           v-on="on"
         >
-          <v-icon v-if="hasApps && (hover || menuOpen)" color="primary"> mdi-apps </v-icon>
-          <img v-else-if="currentApp" :src="base64ImageURI(currentApp.svg_icon)" />
+          <v-icon v-if="hasApps && (hover || menuOpen)" width="100%" color="primary"> mdi-apps </v-icon>
```

Comment:
I think this here wouldn't work. `v-icon` doesn't have a `width` property but a `size`. Do you really need to define a size here. By default Vuetify uses `24px` for the size.

Reply:
👍